raiden-dapp/src/locales/en.json

```diff
@@ -331,5 +331,73 @@
       "rtt": "Ping",
       "price": "Price"
     }
+  },
+  "error-message": {
+    "problem": "Problem:",
+    "solution": "Solution:"
+  },
+  "errors": {
+    "RDN_GENERAL_ERROR": {
```

Comment:
That's the error code for "unknown" errors, e.g. if an `Error` object gets thrown instead of a `RaidenError`